localstack/ext/java/src/main/java/cloud/localstack/lambda/DDBEventParser.java

```diff
@@ -0,0 +1,132 @@
+package cloud.localstack.lambda;
+
+import cloud.localstack.LambdaExecutor;
+import com.amazonaws.services.dynamodbv2.model.*;
+import com.amazonaws.services.lambda.runtime.events.DynamodbEvent;
+
+import java.nio.ByteBuffer;
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+public class DDBEventParser extends LambdaExecutor {
```

Comment:
Since `DDBEventParser` only defines `static` methods and only relies on static methods from `LambdaExecutor`, maybe we should use static imports rather than extending `LambdaExecutor`.